src/components/TeaserGrid/index.js

```diff
@@ -3,21 +3,24 @@ import styles from './teaser-grid.scss';
 import Grid from 'components/Grid';
 import GridItem from 'components/GridItem';
 import Teaser from 'components/Teaser';
+import Loading from 'components/Loading';
 
 const TeaserGrid = ({ petitions, isLoading }) => (
-  <div className={isLoading ? styles.loading : styles.root}>
-    <div className={styles.grid}>
-      <Grid>
-        {(petitions || []).map((petition) => {
-          return (
-            <GridItem key={petition.id}>
-              <Teaser {...petition} key={petition.id} />
-            </GridItem>
-          );
-        })}
-      </Grid>
+  <Loading isLoading={isLoading}>
```

Comment:
I could be up for keeping the `<Grid>` and the `<Loading>` components separate. What do you think?